### DISCUSSION TITLE
be200 wifi7 能驱动了吗？

### DISCUSSION THREAD

**Discussion by 357153645** [istoreos/istoreos#1826] [General]
be200我看到 intel官网有驱动了。但是我安装了istoreos 都找不到WiFi。下载了驱动也不识别。

**Comment by jjm2473** [comment]
先看看openwrt那边有没有驱动，大部分驱动都是openwrt先支持的，少部分才是istoreos移植的。而且intel的wifi基本没卵用，只能做STA模式，不能做AP。
